packages/lisk-p2p/src/peer_pool.ts

```diff
@@ -253,33 +256,79 @@ export class PeerPool extends EventEmitter {
 		});
 	}
 
+	public async fetchStatusAndCreatePeer(
```

Comment:
This method name should be plural because it's fetching and adding multiple peers.